suites/quincy/cephadm/tier-1_service_apply_spec.yaml

```diff
@@ -255,3 +255,8 @@ tests:
               args:              # sleep to get all services deployed
                 - sleep
                 - "120"
+  - test:
+      name: test_label_no_schedule
+      desc: Adding  "__no_schedule" label to a node and verify its functionality in a cluster.
+      polarion-id: CEPH-83574672
+      module: test_label_no_schedule.py
```

Comment:
would it be good to go with particular node to enhance the tests further in future.

```
config:
   nodes: 
      - node1
      - .... 
```

Reply:
Thanks @sunilkumarn417 

Updated